src/config.rs

```diff
@@ -87,6 +87,7 @@ impl Config {
             local_docker_image: maybe_env("DOCS_RS_LOCAL_DOCKER_IMAGE")?,
             toolchain: env("CRATESFYI_TOOLCHAIN", "nightly".to_string())?,
             build_cpu_limit: maybe_env("DOCS_RS_BUILD_CPU_LIMIT")?,
+            only_build_default_target: env("DOCSRS_ONLY_BUILD_DEFAULT_TARGET", false)?,
```

Comment:
```suggestion
            include_default_targets: env("DOCSRS_INCLUDE_DEFAULT_TARGETS", true)?,
```

And an example of `DOCSRS_INCLUDE_DEFAULT_TARGETS=false` in `.env.sample` seems like it'd support defaulting to on without the double-negative of `!only_build_default_target`.

Reply:
@Nemo157 well this is what I mean by I want the env variable to default to `false` - otherwise it's confusing what the default is. If we change it to `INCLUDE_DEFAULT_TARGETS` the default will be true.

Reply:
🤷 I don't find it that confusing when things default to on and you have to explicitly disable them in config, but I don't really care that much.

Reply:
Ok, I changed this to `INCLUDE_DEFAULT_TARGETS`, set it to be on by default in docs.rs, but off by default in env.sample.